remotion/compositions/showcases/mixit2023/Speakers.tsx

```diff
@@ -21,6 +23,17 @@ export const Speakers: React.FC<{speakers: Speaker[]}> = ({speakers}) => {
 		to: 100,
 		durationInFrames: 30,
 	});
+	const titleOpacity = spring({
+		frame: frame - animationDelay * 2,
+		fps,
+		from: 0,
+		to: 1,
+		durationInFrames: 60,
+	});
+
+	const titleDeblur = interpolate(frame - animationDelay * 2, [0, 20], [5, 0], {
```

Comment:
Maybe 'unblur' is more English than 'deblur'

Reply:
C'est plus des title aussi donc peut-être rename ça